lbrynet/daemon/Daemon.py

```diff
@@ -2376,6 +2376,33 @@ def jsonrpc_wallet_send(self, amount, address=None, claim_id=None):
             self.analytics_manager.send_claim_action('new_support')
         defer.returnValue(result)
 
+    @AuthJSONRPCServer.auth_required
+    @defer.inlineCallbacks
+    @AuthJSONRPCServer.flags(no_broadcast='--no_broadcast')
+    def jsonrpc_wallet_prefill_addresses(self, num_addresses, amount, no_broadcast=False):
+        """
+        Create new addresses, each containing `amount` credits
+
+        Usage:
+            wallet_prefill_addresses [--no_broadcast]
+                                     (<num_addresses> | --num_addresses=<num_addresses>)
+                                     (<amount> | --amount=<amount>)
+
+        Returns:
+            (bool) true if addresses were created successfully
```

Comment:
This returns a dict, not a bool.

Reply:
probablly should return a bool to be consistent with other wallet_ commands